src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -295,15 +295,10 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
   }
 
   private renderEntity(entityConf: LovelaceRowConfig): TemplateResult {
-    const element = createRowElement(
-      (!("type" in entityConf) || entityConf.type === "conditional") &&
-        this._config!.state_color
-        ? ({
-            state_color: true,
-            ...(entityConf as EntityConfig),
-          } as EntityConfig)
-        : entityConf
-    );
+    const element = createRowElement({
+      state_color: this._config!.state_color,
```

Comment:
I would say this is not correct, as special case rows don't necessarily support `state_color` and could fail config checks because of it?